.github/workflows/CI.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         with:
           repository: 'opensearch-project/OpenSearch'
           path: OpenSearch
-          ref: 'main'
+          ref: '1.x'
```

Comment:
Why are we changing the branch ref for OpenSearch to `1.x` from `main`? Wouldn't `main` of `anomaly-detection` refer to `main` of OpenSearch and `1.x` of `anomaly-detection` refer to `1.x` of OpenSearch?

Reply:
OpenSearch is already developing a 2.0 on main, and 1.1 on the 1.x branch.

Most plugins don't have a 2.0 line yet, they are doing 1.1 on main.

Reply:
I still see the 1.1.0 version in OpenSearch main: https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/version.properties#L1. Let me know if I am not looking at the right place.

Reply:
That should get updated to 2.0 but it will break the world. We've only be backporting 1.x changes into 1.x.

Reply:
Got it!